aptos-move/framework/aptos-framework/sources/PropertyList.move

```diff
@@ -0,0 +1,182 @@
+/// This Data Structure is designed to hold properties.
+/// It only support
+///     1. adding new property.
+///     2. update the value of existing property
+///     3. get the value of a property
+module AptosFramework::PropertyList {
```

Comment:
We should probably call this PropertyMap, since it is {K:V} and not [(K,V)]

Reply:
Done